src/components/App.js

```diff
@@ -1,20 +1,38 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 
-import Menu from '../containers/MenuContainer.js';
-import Cat from './Cat.js';
-import Message from './Message.js';
-import CallToAction from './CallToAction.js';
-import '../styles/App.css';
+import Menu from "../containers/MenuContainer.js";
+import Cat from "./Cat.js";
+import Message from "./Message.js";
+import CallToAction from "./CallToAction.js";
+import Ball from "./Ball.js";
+import "../styles/App.css";
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isBouncing: false
+    }
+    this.makeBounce = this.makeBounce.bind(this);
+  }
+  makeBounce() {
+    /* HACK ATTACK */
+    if(this.state.isBouncing) { return; }
+    this.setState({ isBouncing: true });
+    window.setTimeout(() => {
```

Comment:
I do not think this is too bad!

The problem here is that if the component unmounts during the anim, you will crash I think.
So you also need to hold onto the timeout hook and cancel in on unmount!